ubcpi/static/html/ubcpi_edit.html

```diff
@@ -0,0 +1,153 @@
+                    </div>
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-correct-rationale">Rationale<span class="tip setting-help">Explain to students why this is the correct option.</span></label>
+                        <textarea name="pi-correct-rationale" id="pi-correct-rationale" ng-model="esc.data.correct_rationale.text" ng-model-options="{ debounce: 500 }" required></textarea>
+                    </div>
+                </li>
+
+                <li class="field comp-setting-entry is-set">
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-algo">Selection Algorithm<span class="tip setting-help">The algorithm for determining which other answers the student answering this question sees after they submit their answer and rationale.</span></label>
+                        <select name="pi-algo" id="pi-algo" ng-model="esc.data.algo.name" ng-options="algo as algo for (algo, desc) in esc.algos" required></select>
+                    </div>
+                    <span class="tip setting-help">{{ esc.algos[esc.data.algo] }}</span>
+                    <div class="wrapper-comp-setting">
+                        <label class="label setting-label pi-setting-label" for="pi-algo-num-responses">Number of Responses<span class="tip setting-help">The number of responses presented to the students. Enter # for the same number as options.</span></label>
+                        <input class="input setting-input pi-options" name="pi-algo-num-responses" id="pi-algo-num-responses" ng-model="esc.data.algo.num_responses" ng-model-options="{ debounce: 500 }" type="text" required />
+                    </div>
+                </li>
+
+                <li class="field comp-setting-entry is-set">
+
+                    <label class="label setting-label" for="">Seeded Answers<span class="tip setting-help">Provide answers which can be shown to the students answering this question when there are not enough submitted responses from which to choose. </span><input type="button" class="button action-primary add-image-button" value="Add Seed" ng-click="esc.addSeed()" name="pi-option-add-seed" /></label>
+
+
+                    <div class="ubcpi-options-list-container">
+                        <div class="wrapper-comp-setting ubcpi-options-list seeded-answer" id="seed-{{ $index + 1 }}" ng-repeat="option in esc.data.seeds track by $index">
+                            <label class="label setting-label pi-setting-label" for="pi-option-seeded-{{$index}}">Seeded Answer {{$index + 1}} <span class="tip setting-help">Choose which option this seeded answer is for</span></label>
```

Comment:
Nit: Helper text on the next line

![screen shot 2015-09-16 at 2 28 55 pm](https://cloud.githubusercontent.com/assets/1985317/9914315/48f78ebe-5c7f-11e5-9b84-b42b7deb417a.png)